index.d.ts

```diff
@@ -0,0 +1,21 @@
+import Koa = require("koa");
```

Comment:
@brunokrebs why would you want to add it to dependencies, it is a TypeScript dependency, which is used only during development phase